### PR TITLE
CORE-8401: use the location sent in the redirect when a redirect is r…

### PR DESCRIPTION
**Pull request by dennisroberts71**
…eturned from an app listing request

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java [hunk 1]
```diff
@@ -77,7 +77,8 @@
         @Override
         public void onFailure(Throwable caught) {
             if (caught instanceof HttpRedirectException) {
-                AgaveAuthPrompt prompt = AgaveAuthPrompt.getInstance();
+                final String uri = ((HttpRedirectException)caught).getLocation();
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java]: Just for my own knowledge - is this redirect still related to Agave?  Is it different from what gets loaded in `userInfo.getAgaveRedirect()`?

**Comment by ash3rz** [comment]
LGTM 🚢

**Comment by dennisroberts71** [comment]
Thanks for the reviews!
